scipy/signal/ltisys.py

```diff
@@ -2494,3 +2496,218 @@ def place_poles(A, B, poles, method="YT", rtol=1e-3, maxiter=30):
     full_state_feedback.nb_iter = nb_iter
 
     return full_state_feedback
+
+
+def controller_Hessenberg(A,B, C=None, tol=None):
+    r"""
+    Reduce the linear system ``AX + BU`` to the controller Hessenberg form
+    H, B_bar where P*A*P.T = H and P*B = Bbar with P being an orthogonnal
```

Comment:
I *think* this will not render properly: asterisks have special meaning for Sphinx. You probably want to either wrap it it in double backticks (i.e. make it render as code) or use `:math:`...`` (so that it's TeX math).